tests/unit_tests/c_wrappers/res/enkf/test_ensemble_config.py

```diff
@@ -10,18 +10,20 @@
 
 
 def test_create():
-    conf = EnsembleConfig()
+    with pytest.raises(ValueError):
+        EnsembleConfig()
+
+    conf = EnsembleConfig.from_dict({})
     assert len(conf) == 0
     assert "XYZ" not in conf
 
     with pytest.raises(KeyError):
-        # pylint: disable=pointless-statement
-        conf["KEY"]
+        _ = conf["KEY"]
```

Comment:
>  do we expect ensemble config to have something under the key, "KEY"

No, I guess that is the point, it tests that if we are trying to retrieve a key in this case confusingly named "KEY" we will get a `KeyError` and the syntax ` conf["KEY"]` is possible only because EnsembleConfig implements:

``` python
    def __getitem__(self, key: str) -> EnkfConfigNode:
        if key in self:
            return self._get_node(key).setParent(self)
        else:
            raise KeyError(f"The key:{key} is not in the ensemble configuration")
```

Reply:
ya of course, thank you. somehow missed the `pytest.raises(KeyError)` :facepalm: